src/services/qrProcessor/index.js

```diff
@@ -0,0 +1,19 @@
+import { get } from "axios";
```

Comment:
cant we use fetch instead ?

Reply:
`fetch` requires the isomorphic fetch import. Perfer having axios over fetch since its api is consistent in node and browser environment and testing is more straightforward. 

You just need `jest.mock("axios");`